### DISCUSSION TITLE
No Release This Week :(

### DISCUSSION THREAD

**Discussion by saddam213** [saddam213/OnnxStack#58] [General]
Sorry team, have been sick all week so version v.0.11.0 will not be packaged and released today :(

Hopefully back to coding tomorrow, after I catch up on my real job (Printing T-Shirts)

Next Fridays v0.12.0 will include all changes and any new development over the next week :)

**Comment by Amin456789** [comment]
u r a genius dude! no matter when u release stuff, onnxstack is great!
